Makefile

```diff
@@ -74,12 +75,19 @@ lint:
 	golint ipamd/*.go
 	golint ipamd/*/*.go
 
-#go tool vet
+# go vet
 vet:
-	go tool vet ./pkg/awsutils
-	go tool vet ./plugins/routed-eni
-	go tool vet ./pkg/k8sapi
-	go tool vet ./pkg/networkutils
+	go vet ./pkg/awsutils
```

Comment:
I noticed in [#413](https://github.com/aws/amazon-vpc-cni-k8s/pull/413/commits/c8faee22a000ca735ae4a69322e11534e753619e#diff-b67911656ef5d18c4ae36cb6741b7965R100) that `GOOS=linux go vet` covers all packages.